webapp/channels/src/components/post_view/embedded_bindings/button_binding/button_binding.tsx

```diff
@@ -118,6 +118,7 @@ class ButtonBinding extends React.PureComponent<Props, State> {
 
         return (
             <button
+                className='btn btn-sm'
                 onClick={this.handleClick}
                 style={customButtonStyle}
```

Comment:
Can we also remove this `customButtonStyle` prop here? It's currently always being passed `undefined`

Reply:
We'll also need to remove line 108